docs/documentation.toml

```diff
@@ -205,7 +205,7 @@ name = "format"
 description = "Format for the date, refer to format reference."
 
 [tp.file.functions.move]
-name = "functions.move"
+name = "file.move"
```

Comment:
Should this be `file.move` or just `move`? The other examples lead me to think it should be `move`